mscore/musescore.cpp

```diff
@@ -7715,6 +7715,8 @@ int runApplication(int& argc, char** av)
             return ok ? EXIT_SUCCESS : EXIT_FAILURE;
             }
 
+      QDir::setCurrent(app->applicationDirPath());
```

Comment:
Please tell me, on windows, if I open score and make Save As, in which default directory will I be prompted to save the file?

Reply:
You will be prompted to save it in the directory containing the opened score file (and not the MuseScore application directory, which I'm guessing is what you were worried about).

This is true in all three of these cases:

1. You have opened the file by double-clicking it in Explorer while MuseScore is not running (thus causing MuseScore to be launched to open the file).
2. You have opened the file by double-clicking it in Explorer while MuseScore is already running (thus causing the file to be opened in the existing instance of MuseScore).
3. You have opened the file by launching MuseScore and going to *File* | *Open*.

I tested these cases before submitting the PR, but I tested them again just now to make sure.